File: dmr-api/Models/Stir.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DMR_API.Models
{
    public class Stir
    {
        public Stir()
        {
            this.CreatedTime = DateTime.Now;
        }
        public int ID { get; set; }
        public string GlueName { get; set; }
        public int? SettingID { get; set; }
        public Setting Setting { get; set; }
        public int RPM { get; set; }

        public int ActualDuration { get; set; }
        public int StandardDuration { get; set; }

        public bool Status { get; set; }
        public double TotalMinutes { get; set; }
        public int MixingInfoID { get; set; }
        public MixingInfo MixingInfo { get; set; }

        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }

        public DateTime CreatedTime { get; set; }

        public DateTime StartScanTime { get; set; }

        public DateTime StartStiringTime { get; set; }
        public DateTime FinishStiringTime { get; set; }
    }
}

File: dmr-app/src/app/_core/_model/stir.ts
import { IGlueType } from './glue-type';

export interface IStir {
    id: number;
    glueName: string;
    glueType: IGlueType;
    settingID: number;
    actualDuration: number;
    standardDuration: number;
    status: boolean;
    mixingInfoID: number;
    startTime: Date;
    endTime: Date;
    createdTime: Date;
    startScanTime: Date;
    startStiringTime: Date;
    finishStiringTime: Date;
}


export interface IStirForAdd {
    id: number;
    glueName: string;
    settingID: number;
    buildingID: number;
    mixingInfoID: number;
    startScanTime: any;
    startStiringTime: any;
}

export interface IStirForUpdate {
    id: number;
    glueName: string;
    settingID: number;
    mixingInfoID: number;
    glueType: IGlueType;
    startScanTime: any;
    buildingID: number;
    finishStiringTime: any;
}
